ES/ESmy020.xml

```diff
@@ -108,97 +431,229 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                      <list>
-                        <item xml:id="a1">
-                           <locus target="#53rb">f. 53rb</locus>
-                           <desc>-vb: Excerpt from the Gospel of John (1 Jn. 1:1-9).</desc>
-                           <quote xml:lang="gez"/>
-                           <quote xml:lang="en"/>
-                           <note> Crudely written, possibly a writing exercise (several words
-                              misspelled). </note>
+
+                       <item xml:id="a1">
+                          <locus from="1rb" to="1vb"/>
+                          <desc>Crude writing execises, possibly a part of <ref target="#a2">Addition 2</msItem>,
+                          followed by a portion of <ref type="work" corresp="LIT5914Hahu">Hahu-syllabary</desc>
                         </item>
+
                         <item xml:id="a2">
-                           <locus target="#71rb">f. 71rb</locus>
-                           <desc>-vb: Excerpt from a liturgical text (an Anaphora).</desc>
-                           <quote xml:lang="gez"/>
-                           <quote xml:lang="en"/>
-                           <note> Crudely written, by the same hand as Additio 1. Possibly a writing
-                              exercise (some words misspelled). </note>
+                           <locus target="#53rb"/>
+                           <desc type="GuestText"><title type="complete" ref="LIT5905FidalaHawarya"/></desc>
+                           <q xml:lang="gez">መልእክተ፡ ዮሐንስ፡ ወልደ፡ ዘብዴዎስ፡ ቀዳምዊ፡ ንዜንወክሙ፡ በእንተ፡ ውእቱ፡</q>
+                           <note>Writing exercise, very crudely written without word dividers. Several words
+                              misspelled.</note>
+                        </item>
+                        <item xml:id="a3">
+                           <locus target="#71rb"/>
+                           <desc>Excerpt from a hymn, very crudely written without word dividers,
+                             partly unreadable.</desc>
+                           <q xml:lang="gez">
+                           ኵሎሙ፡ ሰራዊተ፡ መላእክት፡ መንፈሳዊያን፡ <gap reason="illegible" unit="words" extent="5"/>ግርማሆሙ፡
+                            ወሱራፌል፡ በቅዳሴሆሙ፡ ሰራዊተ፡ መላእክት፡ በበፆታሆሙ፡ <gap reason="omitted" resp="DN"></gap></q>
+                           <q xml:lang="en"/>
+                        </item>
+
+                        <item xml:id="e1">
+                          <locus target="#5r #5v #8v #9r #64v"/>
+                          <desc>Scribbles, very crude writing exercises
+                           in the same secondary hand that wrote <ref target="#a1">Addition 1</ref> and
+                           <ref target="#a2">Addition 2</ref>.</desc>
+                           </item>
+
+                        <item xml:id="e2">
+                          <locus target="#5r"/>
+                           <desc type="findingAid">Note in the upper margin, in <ref target="#h1">Hand 1</hand>,
```

Comment:
close ref by /ref, not by /hand, here and everywhere below

this makes the file invalid